### PR TITLE
Reword some sentences.

### PR DESCRIPTION
**Pull request by FozzieHi**
It can be quite difficult to read some of the sentences, I have reworded them.

### CODE REVIEW

File: app/dashboard/templates/dashboard/recovery_code.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <div class="card-body">
       <h1 class="h3">Recovery codes</h1>
       <p>
-        If you lose access to your authentication device, you can use one of these backup codes to login to your
-        account. Each code may be used only once. Make a copy of these codes, and store it somewhere safe.
+        In the event you lose access to your authenticator app you can use one of these recovery codes to gain access to your account again.
+        Each code may be used only once and you will not be able to view these again, make sure to store them in a safe place.
```
Comment (nguyenkims) [inline app/dashboard/templates/dashboard/recovery_code.html:13]: The current implementation allows user to see the recovery codes again :). This is debatable as some services show the recovery codes only once while others allow user to access the codes again ...

File: app/dashboard/templates/dashboard/setting.html [hunk 2]
```diff
@@ -183,22 +181,22 @@
...
+            To gain additional features and support Simple Login you can upgrade to a Premium plan. <br>
             <a href="{{ url_for('dashboard.pricing') }}" class="btn btn-sm btn-outline-primary">Upgrade</a>
           {% endif %}
         {% elif current_user.lifetime %}
-          You have the lifetime licence.
+          You have lifetime access to the Premium plan.
           {% if not current_user.paid_lifetime %}
             <br>
-            To support SimpleLogin it's possible to change to a paid plan. <br>
+              To gain additional features and support Simple Login you can upgrade to a Premium plan. <br>
```
Comment (nguyenkims) [inline app/dashboard/templates/dashboard/setting.html:195]: I think we can replace `Premium` by `Paid` here as these users are already having the premium license.

File: app/auth/templates/auth/mfa.html [hunk 3]
```diff
@@ -11,18 +11,16 @@
...
 
       <form method="post">
         {{ otp_token_form.csrf_token }}
         <input type="hidden" name="form-name" value="create">
 
         <div class="font-weight-bold mt-5">Token</div>
-        <div class="small-text mb-3">Please enter the 6-digit number displayed in your MFA application
-          (Google Authenticator, Authy, MyDigiPassword, etc) here
-        </div>
+        <div class="small-text mb-3">Please enter the 2FA code from your 2FA authenticator</div>
```
[the remark this answers is not in the document]
Comment (FozzieHi) [reply app/auth/templates/auth/mfa.html:23]: I removed that as on the next page it specifies what applications to use.

File: app/dashboard/templates/dashboard/setting.html (hunk 2 above)
Comment (FozzieHi) [reply app/dashboard/templates/dashboard/setting.html:195]: So this message is for people on a monthly premium plan?

Comment (nguyenkims) [reply app/dashboard/templates/dashboard/setting.html:195]: This message is for users who have the lifetime access *for free* at the beginning :). Some of them have asked if they can contribute back by upgrading to a paid plan so we added this message :)

File: app/auth/templates/auth/mfa.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (nguyenkims) [reply app/auth/templates/auth/mfa.html:23]: If I remember correctly a user needs to enter the code on this page right? After this page, they will be redirected to the dashboard.

[the remark this answers is not in the document]
Comment (FozzieHi) [reply app/auth/templates/auth/mfa.html:23]: Oh right this is after enabling 2FA so they have already set 2FA up using the app before, do we need to remind them what apps they used?

[the remark this answers is not in the document]
Comment (nguyenkims) [reply app/auth/templates/auth/mfa.html:23]: Good point, they don't need to be reminded at this point what app to be used indeed.

**Comment by nguyenkims** [comment]
The PR is perfect, thanks a lot! I'm going to merge it now, it will be deployed today or tomorrow.

**Pull request merged**